lenskit-core/src/main/java/org/lenskit/data/ratings/Rating.java

```diff
@@ -129,7 +153,9 @@ public long getTimestamp() {
      * returns null.
      *  
      * @return {code true} if there is a rating (the preference is non-null).
+     * @deprecated Unrates have gone away.
      */
+    @Deprecated
```

Comment:
Yep. That is my plan. I'm trying to introduce entities with minimal initially-forced changes, then update the algorithms, then rip out the unsupported functionality.